_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -18,12 +18,12 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML `input`, `select` and `textarea` element with a [non-empty](#non-empty) `autocomplete` attribute, except if one of the following is true:
+The rule applies to any HTML `input`, `select` and `textarea` element with a `autocomplete` attribute, that is not the empty string (""), except if one of the following is true:
 
 - The element is not [visible on the page](#visible-on-the-page), and not [included in the accessibility tree](#included-in-the-accessibility-tree)
 - The element is an `input` element with a `type` property of `hidden`, `button`, `submit` or `reset`
 - The element has a `disabled` or `aria-disabled="true"` attribute
-- The element has `tabindex="-1"` and has a [semantic role](#semantic-role) that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles)
+- The element does not participate in [sequential focus navigation](https://www.w3.org/TR/html/editing.html#sec-sequential-focus-navigation) and has a [semantic role](#semantic-role) that is not a [widget](https://www.w3.org/TR/wai-aria-1.1/#widget_roles).
```

Comment:
Sure, it simplifies things, but it's also incorrect. Tab indices are only a fairly small part of determining whether something participates in sequential focus navigation or not.

Reply:
We don't need _all_ elements in sequential navigation, we just need to exclude elements from the applicability that aren't. There are two ways to do that, disable or negative tabindex. Both are covered.

Reply:
...or by having the subtree in which the element resides marked inert due an open `<dialog>`. The point being that by describing what we actually _mean_ rather than indications thereof we hopefully end up with more robust rules.

Reply:
@kasperisager, @WilcoFiers and I have discussed and agreed to go ahead with the "sequential focus navigation".